FlowControllerKit/UIStoryboard-Instantiate.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+extension UIStoryboard {
+
+    /// This is a convenience method that calls the method instantiateViewController(withIdentifier:)
+    /// and derives its identifier from the class name and returns the correct type so no type casting
+    /// is nececery.
+    ///
+    /// The storyboard ID needs to be set to the class name for this to works if not this will throw an exeption.
```

Comment:
The storyboard ID needs to be set to the class name for this to works if not this will throw an exeption.
->
The storyboard ID needs to be set to the class name for this to work, otherwise it'll throw an exception.